<?xml version="1.0" encoding="UTF-8"?>
<testrun name="com.christan.pagetests" project="HttpUnit" tests="4" started="4" failures="2" errors="0" ignored="0">
  <testsuite name="com.christan.pagetests.AboutUsTest" time="8.579">
    <testcase name="testTitle" classname="com.christan.pagetests.AboutUsTest" time="5.829"/>
    <testcase name="testNavigation" classname="com.christan.pagetests.AboutUsTest" time="2.75">
      <failure>java.lang.AssertionError: The file for storing the navigation of this page (navigation_about-us%2F.txt) did not exist previously, creating it on this run.
	at org.junit.Assert.fail(Assert.java:93)
	at com.christan.pagetests.GenericPageTest.testNavigation(GenericPageTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:274)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:48)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:242)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:58)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:240)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:48)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:233)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:303)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:46)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)

</failure>
    </testcase>
  </testsuite>
  <testsuite name="com.christan.pagetests.HomepageTest" time="5.352">
    <testcase name="testTitle" classname="com.christan.pagetests.HomepageTest" time="2.778"/>
    <testcase name="testNavigation" classname="com.christan.pagetests.HomepageTest" time="2.574">
      <failure>java.lang.AssertionError: The file for storing the navigation of this page (navigation_homepage.txt) did not exist previously, creating it on this run.
	at org.junit.Assert.fail(Assert.java:93)
	at com.christan.pagetests.GenericPageTest.testNavigation(GenericPageTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:274)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:48)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:242)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:58)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:240)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:48)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:233)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:303)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:46)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)

</failure>
    </testcase>
  </testsuite>
</testrun>
